evm/deploy-create3-deployer.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+require('dotenv').config();
+
+const { Wallet, getDefaultProvider } = require('ethers');
+const readlineSync = require('readline-sync');
+const { predictContractConstant } = require('@axelar-network/axelar-gmp-sdk-solidity');
+const { Command, Option } = require('commander');
+const chalk = require('chalk');
+
+const { deployCreate2 } = require('./upgradable');
+const { printInfo, writeJSON } = require('./utils');
+const implementationJson = require('../artifacts/contracts/deploy/Create3Deployer.sol/Create3Deployer.json');
+const contractName = 'Create3Deployer';
+
+async function deploy(options, chain) {
+    const { env, privateKey, verify } = options;
+    const wallet = new Wallet(privateKey);
+
+    printInfo('Deployer address', wallet.address);
+
+    const rpc = chain.rpc;
+    const provider = getDefaultProvider(rpc);
+
+    console.log(
+        `Deployer has ${(await provider.getBalance(wallet.address)) / 1e18} ${chalk.green(
+            chain.tokenSymbol,
+        )} and nonce ${await provider.getTransactionCount(wallet.address)} on ${chain.name}.`,
```

Comment:
not urgent but can also be refactored. in particular, each chain config can have contract specific or chain specific gas options. We could sanity check if the balance covers the gas limit * gas price or so. can make a note to tackle this later

Reply:
will make a note